Source/EasyNetQ/Persistent/PersistentConnectionState.cs

```diff
@@ -0,0 +1,11 @@
+namespace EasyNetQ.Persistent;
+
+/// <summary>
+///     A state of connection
+/// </summary>
+public enum PersistentConnectionState
+{
+    Unknown,
```

Comment:
I thought initial state is known and it is something like "NotConnected" but not unknown

Reply:
Changed to NotInitialised. NotConnected is very similar to Disconnected.